openfisca_us/tests/policy/baseline/hhs/tanf/tanf_amount_if_eligible.yaml

```diff
@@ -0,0 +1,59 @@
+- name: "TANF: Has given maximum benefit amount and has smaller countable income"
+  period: 2022
+  absolute_error_margin: 0.01
+  input:
+    tanf_max_amount: 1000
+    tanf_countable_income: 750
+  output:
+    tanf_amount_if_eligible: 1000 - 750
+
+- name: "TANF: Has given maximum benefit amount and has larger countable income"
+  period: 2022
+  absolute_error_margin: 0.01
```

Comment:
Does OpenFisca have some small rounding issue here? Seems inconsequential but just curious

Reply:
It did for some other test so I had to relax it in https://github.com/PolicyEngine/openfisca-us/commit/b99f2c46ef5ecce768658d6f8e0a8ddef20574ed